frappe/patches/v14_0/update_attachment_comment.py

```diff
@@ -0,0 +1,31 @@
+import frappe
+
+
+def execute():
+	# Strip everything except link to attachment and icon from comments of type "Attached"
+	for name, content in frappe.get_all(
+		"Comment", filters={"comment_type": "Attachment"}, fields=["name", "content"], as_list=True
+	):
+		if not content:
+			continue
+
+		start = content.find("<a href")
+		if start != -1:
+			content = content[start:]
+
+		end = content.find("</i>")
+		end = content.find("</a>") if end == -1 else end
+		if end != -1:
+			content = content[: end + 4]
```

Comment:
Instead of doing this why not just strip leading `"Added "` like you've done for removed, it will be simpler and reliable? 😅

Reply:
AH is it because it was previously translated?